.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,71 @@
+name: Bug Report
+description: I experienced unexpected behavior using useSimpleReducer.
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: Found a bug? Thanks for taking the time to report it and help make useSimpleReducer better! Fill out the form below to help our team get started.
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Environment
+        What environments were you in when you encoutnered this bug?
+
+  - type: input
+    attributes:
+      label: Node
+      description: What version of Node?
+      placeholder: '16'
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: React
+      description: What version of React?
+      placeholder: '17'
+    validations:
+      required: true
+
```

Comment:
What meta-frameworks and build systems are you using, and their versions? NextJS, Gatsby, Vite, etc.